### PR TITLE
Simon/82 departure marker

### PR DESCRIPTION
**Pull request by SimonChow9527**
[AB#82](https://dev.azure.com/Tringo/cabcbfb6-3a22-44cf-ac98-b3810925a870/_workitems/edit/82)

currently back-end won't return lat and lng for departure airport, so hardcoded sydney for now.

### CODE REVIEW

**Comment by rmit-s3668804-Dzmitry-Kakaruk** [comment]
straight away no hardcoded departure. we are passed that moment now.

**Review by rmit-s3668804-Dzmitry-Kakaruk — CHANGES_REQUESTED** [review]
refactor to dynamically set location

**Comment by SimonChow9527** [comment]
how could I do that when back-end doesn't provide the values I need? use env?

**Comment by rmit-s3668804-Dzmitry-Kakaruk** [comment]
you should’ve raised this issue in dev channel and created a separate card because there’s a blocker for you. we had standup of wednesday this is i where you raise your blockers

File: app/src/components/googleMap.tsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 import SearchWidgetWrapper from 'components/searchWidget/searchWidgetWrapper';
-import { FlightDestinationRequest, MapArea } from 'models/request/flightDestinationRequest';
-import { DatesInput } from 'models/request/dateInput';
+import {FlightDestinationRequest, MapArea} from 'models/request/flightDestinationRequest';
+import {DatesInput} from 'models/request/dateInput';
 import gMapConf from './gMapConf.json';
-import { DestinationsState } from 'models/response/destinations';
-import { withStyles, LinearProgress } from '@material-ui/core';
+import {DestinationsState} from 'models/response/destinations';
+import {withStyles, LinearProgress} from '@material-ui/core';
```
Comment (s3419069) [inline app/src/components/googleMap.tsx]: again... what's wrong with your IDE? please set up formatting

File: app/src/components/googleMap.tsx [hunk 2]
```diff
@@ -138,6 +140,18 @@ class SimpleMap extends React.Component<MapProp, MapState> {
         });
     }
 
+    renderDepartureAirport() {
+        // hardcoded Sydney for now
+        if (true) {
+
+            return (<DepartureMarker
```
Comment (s3419069) [inline app/src/components/googleMap.tsx]: one more formatting issue

File: app/src/components/markers/tinyPinMarker.scss [hunk 3]
```diff
@@ -3,16 +3,17 @@
...
 $markerSizehovered: 18px;
 .tiny-marker {
-    width: $markerSize;
-    height: $markerSize;
-    border-radius: 100%;
-    background: $lightBlue;
-    margin-top: -2px;
-    margin-left: -3px;
-    border: 2px solid $lighterGrey;
+  width: $markerSize;
```
Comment (s3419069) [inline app/src/components/markers/tinyPinMarker.scss:6]: please set in your IDE tab as 4 spaces, not 2

Comment (SimonChow9527) [reply app/src/components/markers/tinyPinMarker.scss:6]: sure

**Comment by SimonChow9527** [comment]
departure
now looks like this

**Comment by s3419069** [comment]
1. Why is it in the middle of the sea?? Do **not** use standard Google's marker. We need to have custom nice one, with City name inside preferably.

2. Do **not** use props object from priceTagMarker. it never has a price and many other attributes.

3. In File changed tab I and Dimi want to see changes related **ONLY** to this PBI (exclude all other formatting changes)

**Pull request closed without merging**